CMakeLists.txt

```diff
@@ -231,11 +232,14 @@ target_link_libraries( testclient ${QTKEYCHAIN_TARGET_NAME})
 
 configure_file(QtKeychainBuildTreeSettings.cmake.in
   "${PROJECT_BINARY_DIR}/Qt${QTKEYCHAIN_VERSION_INFIX}KeychainBuildTreeSettings.cmake" @ONLY)
-configure_file(QtKeychainConfig.cmake.in
-  "${PROJECT_BINARY_DIR}/Qt${QTKEYCHAIN_VERSION_INFIX}KeychainConfig.cmake" @ONLY)
 configure_file(QtKeychainConfigVersion.cmake.in
```

Comment:
FYI: There's also `ecm_setup_version` which helps generating the ConfigVersion, too.